modules/EasyAbp.EShop.Orders/src/EasyAbp.EShop.Orders.Application/EasyAbp/EShop/Orders/Orders/OrderAppService.cs

```diff
@@ -149,24 +148,20 @@ public virtual async Task<OrderDto> GetByOrderNumberAsync(string orderNumber)
 
             if (order.CustomerUserId != CurrentUser.GetId())
             {
-                await AuthorizationService.CheckAsync(OrdersPermissions.Orders.Manage);
-
-                // Todo: Check if current user is an admin of the store.
+                await AuthorizationService.CheckStoreOwnerAsync(order.StoreId, OrdersPermissions.Orders.Manage);
```

Comment:
I see an AuthorizationHandler below, but the idea of this comment still has reference significance. (keep the PR lightweight since a huge PR is really hard to review)